hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -363,6 +363,32 @@
         }
       ]
     },
+    { name: "MARKOV_LO_THRESHOLDS",
+      desc: "Markov test low thresholds register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen:   "REGEN",
```

Comment:
do you think there's any reason these registers would not all be programmed at the same time?
since there's only one `REGEN`, it would imply most things would have to be locked at one time right?

That might be perfectly fine, but I just wanted to check.

Reply:
I would think that, but how firmware controls the timing of setup and lockdown is really out of our scope. All we can do is provide the base facilities and make they they work properly. At least from the various security meetings that is my take away.